src/backend/common/graphics_common.hpp

```diff
@@ -93,14 +101,10 @@ class ForgeManager
         WindGridMap_t       mWndGridMap;
         ChartAxesOverride_t mChartAxesOverrideMap;
 
-        ForgeModule* mPlugin;
-
     public:
-        static ForgeManager& getInstance();
+        ForgeManager();
```

Comment:
May be in general, but I don't think we need that here. You may not have noticed that constructor, copy-constructor and copy-assignement-operator of ForgeManager were protected earlier. We always have had only one instance of ForgeManager, earlier or now. In this PR, I just made the constructor public so that the `DeviceManager` singleton can create a member instance of `ForgeManager`. The ForgeManager instance still cannot be copied anywhere else. 

We definitely don't want to move this instance either, so I can add move constructors to be protected as well.